services/terraformer/templates/oci.tpl

```diff
@@ -0,0 +1,153 @@
+{{- $clusterName := .ClusterName}}
+{{- $clusterHash := .ClusterHash}}
+{{$index :=  0}}
+
+variable "default_compartment_id" {
+  type = string
+  default = "{{(index .NodePools 0).Provider.CompartmentOcid}}"
+}
+
+provider "oci" {
+  tenancy_ocid = "{{(index .NodePools 0).Provider.TenancyOcid}}"
+  user_ocid = "{{(index .NodePools 0).Provider.UserOcid}}"
+  fingerprint = "{{(index .NodePools 0).Provider.OciFingerprint}}"
+  private_key_path = "{{(index .NodePools 0).Provider.SpecName}}" 
+  region = "{{(index .NodePools 0).Region}}"
+  alias = "k8s-nodepool"
+}
+
+resource "oci_core_vcn" "claudie_vcn" {
+  provider = oci.k8s-nodepool
+  compartment_id = var.default_compartment_id
+  display_name = "{{ $clusterName }}-{{ $clusterHash }}-vcn"
+  cidr_blocks = ["10.0.0.0/16"]
+}
+
+resource "oci_core_subnet" "claudie_subnet" {
+  provider = oci.k8s-nodepool
+  vcn_id = oci_core_vcn.claudie_vcn.id
+  cidr_block = "10.0.0.0/24"
+  compartment_id = var.default_compartment_id
+  display_name = "{{ $clusterName }}-{{ $clusterHash }}-subnet"
+  security_list_ids = [oci_core_vcn.claudie_vcn.default_security_list_id]
+  route_table_id = oci_core_vcn.claudie_vcn.default_route_table_id
+  dhcp_options_id   = oci_core_vcn.claudie_vcn.default_dhcp_options_id
+}
+
+resource "oci_core_internet_gateway" "claudie_gateway" {
+  provider = oci.k8s-nodepool
+  compartment_id = var.default_compartment_id
+  display_name   = "{{ $clusterName }}-{{ $clusterHash }}-gateway"
+  vcn_id         = oci_core_vcn.claudie_vcn.id
+  enabled = true
+}  
+
+resource "oci_core_default_security_list" "claudie_security_rules" {
```

Comment:
Is there any difference between [oci_core_security_lists](https://docs.oracle.com/en-us/iaas/tools/terraform-provider-oci/4.95/docs/d/core_security_lists.html) and `oci_core_default_security_list` ?

Reply:
The `oci_core_default_security_list` is used to configure the default resource created for every VCN, whereas `oci_core_security_list` defines new security list.